mne_bids/sidecar_updates.py

```diff
@@ -136,3 +144,35 @@ def _update_sidecar(sidecar_fname, key, val):
     sidecar_json[key] = val
     with open(sidecar_fname, 'w', encoding='utf-8') as fout:
         json.dump(sidecar_json, fout)
+
+
+def update_anat_landmarks(bids_path, info, trans, t1w=None):
```

Comment:
wait for https://github.com/mne-tools/mne-bids/pull/827

so you just pass the landmarks (do not pass info and trans) it's buggy

Reply:
ok